metricbeat/module/kubernetes/_meta/config.yml

```diff
@@ -21,3 +21,10 @@
     - state_container
   period: 10s
   hosts: ["kube-state-metrics:8080"]
+
+# Kubernetes events
+- module: kubernetes
+  metricsets:
+    - event
```

Comment:
I think event (or events) is ok in this case, this is something unrelated to `kube-state-metrics`, state_ metricsets get their name from it